.github/workflows/build.yml

```diff
@@ -218,6 +233,21 @@ jobs:
       arch: ${{ matrix.arch }}
       free-threading: ${{ matrix.free-threading }}
 
+  build_windows_msi:
+    name: >-  # ${{ '' } is a hack to nest jobs under the same sidebar category
+      Windows MSI${{ '' }}
```

Comment:
@webknjaz Someone else has said "Tests / Windows MSI${{ '' }} (pull_request)" looks like a bug.

I think in some other thread you said we can adjust it somehow, is that possible?

Reply:
@hugovk basically, there are two options:
* split this into two matrices in the top-level workflow
* move the interpolated part into the reusable ones

I personally like the second option. Let me know if you or @ambv want something different.

Reply:
Though, I think both might break grouping 🤔

Reply:
Hmm, any other ideas? Someone else has asked about it because it looks weird.

Reply:
I think it would be most impactful if GitHub fixed this in its own UI rather than having people resort to hacks. I'd be happy to drop the `${{ '' }}` hack from other places.

Would it be less confusing is we switched `Windows MSI${{ '' }}` to `${{ 'Windows MSI' }}`?

Reply:
How would that look for the different builds?

Reply:
The same, just bits of syntax would be around the string, not after.

Reply:
Yeah, I think that might be a bit better?

The recent query was actually about the `fromJSON` stuff, do you think we can do anything about that?

![image](https://github.com/user-attachments/assets/e42a22fe-cc7c-4fc1-a7c1-62a43ad0be3b)

Reply:
Duplicating the matrix or moving the conditional inside the reusable workflow would improve this, I suppose.

Reply:
I've done some thinking and realized that duplicating the matrices is the only way to make it look different while preserving the matrix grouping in the UI.